GameServerLib/Config.cs

```diff
@@ -129,7 +129,12 @@ private string GetContentPath()
             return result;
         }
     }
-
+    public class GameFeatures : IGameFeatures
+    {
+        public bool CooldownsEnabled { get; set; }
```

Comment:
Same as above, change to `private set` and utilize a function to set.
```suggestion
        public bool CooldownsEnabled { get; private set; }
        public bool ManaCostsEnabled { get; private set; }
        public bool MinionSpawnsEnabled { get; private set; }
```